### PR TITLE
fix(manager): add length to field array meta

### PR DESCRIPTION
**Pull request by Hyperkid123**
part of https://github.com/data-driven-forms/react-forms/issues/721

### CODE REVIEW

File: packages/form-state-manager/src/files/use-field-array.ts [hunk 1]
```diff
@@ -79,7 +79,7 @@ const useFieldArray: UseFieldArray = ({ name, initialValue }) => {
   };
   return {
     fields,
-    meta
+    meta: { ...meta, length: internalValue.length }
```
Comment (rvsia) [inline packages/form-state-manager/src/files/use-field-array.ts:82]: according to docs https://final-form.org/docs/final-form/types/FieldState#length

It should be for all fields.

Comment (Hyperkid123) [reply packages/form-state-manager/src/files/use-field-array.ts:82]: Well, that an `useField` attribute then. And affects non-field array fields as well.

Comment (rvsia) [reply packages/form-state-manager/src/files/use-field-array.ts:82]: It should be calculated in managerApi.

Comment (Hyperkid123) [reply packages/form-state-manager/src/files/use-field-array.ts:82]: Yep

Comment (Hyperkid123) [reply packages/form-state-manager/src/files/use-field-array.ts:82]: We should fix it in another issue

**Review by rvsia — APPROVED** [review]

**Pull request merged**